Makefile

```diff
@@ -22,7 +25,7 @@ ocp_run:
 	./06_deploy_bootstrap_vm.sh
 	./07_deploy_masters.sh
 
-clean: masters_cleanup ocp_cleanup host_cleanup
+clean: prereq masters_cleanup ocp_cleanup host_cleanup
```

Comment:
Why do we need to add it to clean?

Reply:
Just in case. It doesn't hurt, but I'll remove it if needed.

Reply:
Yeah we don't need to check prerequisites in clean IMHO, it's a pre-deploy check

Reply:
I'll remove it, thanks!